test/unittest/layers/layers_common_tests.h

```diff
@@ -79,11 +79,24 @@ class LayerSemantics
   bool must_fail;
 };
 
+typedef enum {
+  SKIP_CALC_GRAD = 1 << 0,  /**< skip calculating gradient and compare */
+  SKIP_CALC_DERIV = 1 << 1, /**< skip calculating derivative and compare */
+
+  FORWARD_MODE_INFERENCE =
+    1 << 2, /**< set if layer should be forwarded with inference mode */
+
+  DEFAULT = 0, /**< default set up */
+  ONLY_FORWARD =
```

Comment:
ONLY_FORWARD -> FORWARD_TRAIN_MODE?

Reply:
It's bit flag for presetting SKIP_CALC_GRAD, SKIP_CALC_DERIV, 

I think, FORWARD_TRAIN_MODE | FORWARD_MODE_INFERENCE would be hard to infer.

Reply:
ONLY_FORWARD should clarify if it is train or inference, which it is currently not clarifying.

Reply:
I mean, It is intentionally not clarifying the mode because it doesn't care. It's bit flag preset. It's not setting any bit to the mode.

Reply:
I meant not from the perspective of bit flag or bit mode but for readability.

Reply:
Well, this is used like https://github.com/nnstreamer/nntrainer/pull/1587/files#diff-c7e99e86961362f866b90c337b8af3b91a70b8f84a84bb5e0f386d81f0a619a2R26-R27 this. 

Now it will become `bn_inference_option = FORWARD_TRAIN_MODE | FORWARD_MODE_INFERENCE` which looks not intuitive IMO.

Reply:
How about deleting this if this preset seems to cause some misunderstanding?

Reply:
Done